
Example reproduces error when Py_Initialize() is called again after Py_Finalize()

make all
make -B all

./app

make clean